srml/executive/src/lib.rs

```diff
@@ -259,6 +266,7 @@ impl<
 
 		// Decode parameters and dispatch
 		let (f, s) = xt.deconstruct();
+		// TODO TODO: does a unsigned transaction validated by module having Inherent is correct ?
```

Comment:
Can I say because the call is veryfied by a module the call itself is inherently signed ?
that seems to be a valid use of the inherent word for me but well it can confuse with other inherent such as timestamp.

Reply:
No it is not an inherent origin. But it is not interesting to distinguish from it. So inherent should be renamed to nobody or something like that.